geonode/security/utils.py

```diff
@@ -56,7 +56,7 @@ def get_visible_resources(
     admin_approval_required=False,
     unpublished_not_visible=False,
     private_groups_not_visibile=False,
-    include_dirty=False,
+    include_dirty=True,
```

Comment:
i think also the util should use the True by default [get_visible_resources](https://github.com/GeoNode/geonode/blob/bde9c9dadcb79d97ccc17254a917cca9ec837866/geonode/security/utils.py#L59)